### PR TITLE
Create .gitattributes

### PR DESCRIPTION
**Pull request by imbsky**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Auto detect text files and perform LF normalization
+# https://www.davidlaing.com/2012/09/19/customise-your-gitattributes-to-become-a-git-ninja/
+* text=auto
+
+# Linguist override
+*.re linguist-language=Reason
+*.rei linguist-language=Reason
```
Comment (MoOx) [inline .gitattributes]: I think this part is now unecessary. A few week GitHub added auto detection of Reason language.

Not sure about adding a file for the text=auto thing. I was about to delete all .gitattributes from all repos of this org.

What do you think?

Comment (imbsky) [reply .gitattributes]: No, Reason is still misdetected. The change is just to remove Reason from the OCaml group, and the root cause of the problem is that GitHub still supports the rarely used re extension for C++.

**Comment by imbsky** [comment]
In other words, it's a config for the conversion of newline characters, and to some extent a matter of taste. I didn't have a very strong opinion about that so I removed it.

**Comment by MoOx** [comment]
Thanks again!

**Pull request merged**